{{ cookiecutter.package_name }}/.travis.yml

```diff
@@ -92,6 +92,7 @@ matrix:
         # may run for a long time
         - os: linux
           env: SETUP_CMD='build_docs -w'
+               CONDA_DEPENDENCIES='sphinx-astropy{% if cookiecutter.include_example_cython_code == 'y' %} Cython{% endif %}'
```

Comment:
I think it may be a better approach to define a `CONDA_DEPENDENCIES_DOC` above, and use it here otherwise I expect issues that maintainers need to remember to update the dependencies in 2 distinct places of the config.

Reply:
Done!